### PR TITLE
Website custom 404 page

### PR DESCRIPTION
**Pull request by kokopelli314**
Fixes [#1682](https://github.com/Khan/KaTeX/issues/1682). Implements one custom 404 page, but Netlify also allows you to make different pages by language, etc.

Happy to make any changes to add/remove content, styling, or math jokes. 😁

### CODE REVIEW

**Comment by kevinbarabash** [comment]
Thanks for the PR.  I really like the concept.  I'm have a couple of suggestions though.  Instead of "Page Not Found", maybe it could say something along the lines: "Sorry, we couldn't find the page you were looking for.  Try one of these links below:".  Also, the `\lim_{x \leftarrow 404}...` part could be a lot bigger to fill the space.  Also the text and the math should probably be off blank instead of pure black, maybe #222222.  Here's my attempt at a mock (I wasn't able to update the math color because I was using Preview.app to make these changes):

screen shot 2018-10-20 at 3 25 17 pm

**Comment by kokopelli314** [comment]
Thanks! I've just updated the styles and text similar to your screenshot, with #222 text color. For small screens, the equation will be smaller.

I also made the references to JS and CSS resources absolute (prepending `/` to the `static` path), so that a 404 HTTP status can be returned along with the HTML page, while not breaking for long paths (such as `katex.org/doesnt/exist/sub/path`).

File: .browserslistrc [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+last 2 versions
+not dead
+
+# Browsers with more than 0.1% global usage, excluding old browsers
+> 0.1%
+not chrome < 45
+not ie < 9
+
+# Currently maintained Node.js versions
+maintained node versions
```
Comment (kevinbarabash) [inline .browserslistrc]: @kokopelli314 what changed in this file?

Comment (kokopelli314) [reply .browserslistrc]: Looks like it was just a formatting issue. Reverted the file.

**Comment by kokopelli314** [comment]
Quite odd, after reverting the file back to its original state, the deploy/netlify step failed. Looking through the Git blame, I'm not seeing any changes from what I previously had to the original commit, either.

File: .browserslistrc (hunk 1 above)
Comment (ylemkimon) [reply .browserslistrc]: Sorry, this is probably on me 😢 I use Windows and have created file with CRLF line endings.

File: website/static/static/404.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+body {
+    background: #329894;
+}
+
+.main {
+    display: flex;
```
Comment (ylemkimon) [inline website/static/static/404.css]:
```suggestion
    display: -ms-flexbox;
    display: flex;
```

File: website/static/static/404.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    flex-direction: column;
```
Comment (ylemkimon) [inline website/static/static/404.css]:
```suggestion
    -ms-flex-direction: column;
    flex-direction: column;
```

File: website/static/static/404.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    justify-content: space-between;
```
Comment (ylemkimon) [inline website/static/static/404.css]:
```suggestion
    -ms-flex-pack: justify;
    justify-content: space-between;
```

**Comment by kevinbarabash** [comment]
Long term I wonder if we should add some sort of build step for the CSS to auto prefix stuff.

**Comment by kevinbarabash** [comment]
> Quite odd, after reverting the file back to its original state, the deploy/netlify step failed. Looking through the Git blame, I'm not seeing any changes from what I previously had to the original commit, either.

Not sure what was going on there but I don't think it was anything you did.

**Pull request merged**